package/skylark_upload_daemon/Config.in

```diff
@@ -0,0 +1,7 @@
+config BR2_PACKAGE_SKYLARK_UPLOAD_DAEMON
+	bool "skylark_upload_daemon"
+	select BR2_PACKAGE_LIBCURL
+	select BR2_PACKAGE_CZMQ
```

Comment:
TODO: WTF formatting
TODO: WTF how does this even work? It needs to depend on libskylark :/